src/game_interpreter.cpp

```diff
@@ -3456,39 +3610,132 @@ bool Game_Interpreter::CommandToggleFullscreen(lcf::rpg::EventCommand const& /*
 	return true;
 }
 
-bool Game_Interpreter::CommandManiacGetSaveInfo(lcf::rpg::EventCommand const&) {
+bool Game_Interpreter::CommandManiacGetSaveInfo(lcf::rpg::EventCommand const& com) {
 	if (!Player::IsPatchManiac()) {
 		return true;
 	}
 
-	Output::Warning("Maniac Patch: Command GetSaveInfo not supported");
-	return true;
-}
+	int save_number = ValueOrVariable(com.parameters[0], com.parameters[1]);
 
-bool Game_Interpreter::CommandManiacSave(lcf::rpg::EventCommand const&) {
-	if (!Player::IsPatchManiac()) {
+	// Error case, set to YYMMDD later on success
+	Main_Data::game_variables->Set(com.parameters[2], 0);
+
+	if (save_number <= 0) {
+		Output::Debug("ManiacGetSaveInfo: Invalid save number {}", save_number);
+		return true;
+	}
+
+	auto savefs = FileFinder::Save();
+	std::string save_name = Scene_Save::GetSaveFilename(savefs, save_number);
+	auto save = lcf::LSD_Reader::Load(save_name, Player::encoding);
+
+	if (!save) {
+		Output::Debug("ManiacGetSaveInfo: Save not found {}", save_number);
 		return true;
 	}
 
-	Output::Warning("Maniac Patch: Command Save not supported");
+	std::time_t t = lcf::LSD_Reader::ToUnixTimestamp(save->title.timestamp);
+	std::tm* tm = std::gmtime(&t);
+
+	// YYMMDD
```

Comment:
Since this is repeated (return `YYMMDD`/`HHMMSS` as integer from time), might move to common function? (in utils?)

Reply:
replaced the calculation with ``strftime``, much simpler this way ;)

Reply:
Thought about it too, however needs ugly conversion this way.